benchmark/feldera-sql/run.py

```diff
@@ -181,65 +160,21 @@ def main():
     min_storage_bytes = parser.parse_args().min_storage_bytes
     if min_storage_bytes is not None:
         min_storage_bytes = int(min_storage_bytes)
-    suffix = parser.parse_args().input_topic_suffix or ''
     csvfile = parser.parse_args().csv
     csvmetricsfile = parser.parse_args().csv_metrics
     metricsinterval = float(parser.parse_args().metrics_interval)
 
-    output_connector_names = queries
-    if merge and len(queries) > 1:
-        merged_name = ','.join(queries)
-        QUERY_SQL[merged_name] = '\n'.join([QUERY_SQL[q] for q in queries])
-        queries = [merged_name]
-
     when = time.strftime('%Y-%m-%d %H:%M:%S', time.gmtime(time.time()))
 
+    print("Creating programs...")
     for program_name in queries:
         # Create program
         full_name = get_full_name(folder, program_name)
         program_sql = table + all_queries[program_name]
-        response = requests.put(f"{api_url}/v0/programs/{full_name}", headers=headers, json={
+        requests.put(f"{api_url}/v0/pipelines/{full_name}", headers=headers, json={
```

Comment:
still using the low level API?

Reply:
It works and we have a release in an hour.